README.rst

```diff
@@ -1,6 +1,22 @@
-=====================
-SageMaker Experiments
-=====================
+.. image:: https://github.com/aws/sagemaker-experiments/raw/master/branding/icon/sagemaker-banner.png
```

Comment:
Page not found.

Reply:
the banner is part of this pull request see https://github.com/aws/sagemaker-experiments/pull/20/commits/a67bcb644e94d9354fb21025612311aa58854f4e